Makefile

```diff
@@ -217,6 +253,13 @@ test-import:
 test-rpc:
 	./scripts/integration-test-all.sh -q 1 -z 1 -s 2
 
+test-contract:
+	@type "npm" 2> /dev/null || (echo 'Npm does not exist. Please install node.js and npm."' && exit 1)
+	@type "solcjs" 2> /dev/null || (echo 'Solc does not exist. Please install solc."' && exit 1)
```

Comment:
so according to the [solc](https://www.npmjs.com/package/solc) package, it installs `solcjs`? but the contract requires `solc`? I'm a bit confused by that

Reply:
very confused why you still have that 
<img width="766" alt="Screen Shot 2020-09-03 at 9 33 18 AM" src="https://user-images.githubusercontent.com/13338103/92142354-8ba04500-edc8-11ea-9ace-d161dd4af12c.png">

Reply:
oof, i think something happened and it reverted back to the "old" files.